MerlinAU.sh

```diff
@@ -4091,7 +4148,7 @@ Please manually update to version $minimum_supported_version or higher to use th
        ! _CreateDirectory_ "$FW_BIN_DIR" ; then return 1 ; fi
 
     # Get current firmware version #
-    current_version="$(_GetCurrentFWInstalledShortVersion_)"
+    current_version="$(_GetCurrentFWInstalledLongVersion_)"
```

Comment:
Approved the changes to long version

Reply:
With all the changes to use the long version, do we really need the short version much anymore? I'm assuming it can be removed correct?